### DISCUSSION TITLE
Best sensor to check if PC is on or off?

### DISCUSSION THREAD

**Discussion by Sungod3000** [sleevezipper/hass-workstation-service#178] [Q&A]
Hi, im loving hass workstation so far. Im using it with node red to shut down my PC and the works really well. 

My PC and soundsystem is on one smart switch that I shut down every night. 
However cant really find a sensor to check if the PC is off after I shut it down. 

I tried to use CPU speed but when the PC is off the sensor reads and "NaN" and that I cant use further without having to write a function that reinterprets that output. 
I also tried the session state, but that reads as "unknown" or "unavailable" which is also not easily used as a trigger. 

I wonder if im thinking to complicated here? How would you guys go about checking if the PC is off?

Cheers F

**Comment by OskarKlintrot** [comment]
Did you figure something out? I'm having the same issue.

**Comment by Sungod3000** [comment]
hi @OskarKlintrot kind of. i read somewhere that you can use a dummy sensor that refresehs every 20 secs or so. if the pc turns off the sensor goes "unavailable". and that in turn can be used as a trigger in HA
